docs/support-bundle-guide.md

```diff
@@ -0,0 +1,255 @@
+including logs, so please review the contents before sharing it on Github
+and ensure that you do not share any sensitive information.**
+
+There are two ways of generating support bundles. Firstly, you can run `antctl supportbundle`
+directly in the Antrea Agent Pod, Antrea Controller Pod, or on a host with a
+`kubeconfig` file for the target the cluster. Secondly, you can also apply
+`SupportBundleCollection` CRs to create support bundles for K8s Nodes
+or external Nodes. We name this feature as `SupportBundleCollection` in Antrea.
+The details are provided in section [Usage examples](#usage-examples).
+
+## Table of Contents
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [The SupportBundleCollection CRD](#the-supportbundlecollection-crd)
+- [Usage examples](#usage-examples)
+  - [Running antctl commands](#running-antctl-commands)
+  - [Applying SupportBundleCollection CR](#applying-supportbundlecollection-cr)
+- [List of collected items](#list-of-collected-items)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## Prerequisites
+
+The `antctl supportbundle` command is supported in Antrea since version 0.7.0.
+
+The `SupportBundleCollection` CRD is introduced in Antrea v1.10.0 as an alpha
+feature. The feature gate must be enabled in both antrea-controller and
+antrea-agent configurations. If you plan to collect support bundle on an external
+Node, you should enable it in the configuration on the external Node as well.
+
+```yaml
+  antrea-agent.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+A single Namespace (e.g., default) is created for saving the Secrets that are
+used to access the support bundle file server, and the permission to read Secrets
+in this Namespace is given to antrea-controller by modifying and applying the
+[RBAC file](/build/yamls/externalnode/support-bundle-collection-rbac.yml).
+
+```yaml
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antrea-read-secrets
+  namespace: default  # Change the Namespace to where the Secret for file server's authentication credential is created.
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: antrea-secret-reader
+subjects:
+  - kind: ServiceAccount
+    name: antrea-controller
+    namespace: kube-system
+```
+
+## The SupportBundleCollection CRD
+
+SupportBundleCollection CRD is introduced to supplement the `antctl` command
+with three additional features:
+
+1. Allow users to collect support bundle files on external Nodes.
+2. Upload all the support bundle files into a user-provided SFTP Server.
+3. Support tracking status of a SupportBundleCollection CR.
+
+Details are provided in the next section.
```

Comment:
This seems too verbose if it just points to the next section.

Reply:
removed this sentence.